app/javascript/components/HomePageContent.js

```diff
@@ -29,11 +31,11 @@ const LinkableSearchTabs = function(props) {
     <div>
       <nav className="nav search-links">
         <Link to={`/single_cell/app/studies${location.search}`}
-              className={isShowGenes ? '' : 'active'}>
+          className={isShowGenes ? '' : 'active'}>
```

Comment:
The name `isShowGenes` seems slightly off -- maybe it's that we have no element "Show genes".  Perhaps clearer: `showGeneSearch` or `isGeneSearch`.  Matter of opinion; feel free to ignore.

Reply:
concur